unit_tests/test_base_version.py

```diff
@@ -112,6 +112,14 @@ def test_2025_1_release_ubuntu():
     assert {'6.2', '2024.1', '2024.2'}.issubset(set(version_list))
 
 
+def test_2025_1_release_rocky():
+    """ Test that 2025.1 release on centos is returned correct versions """
+    scylla_repo = url_base + '/branch-2025.4/rpm/centos/2025-02-23T16:19:08Z/scylla.repo'
+    linux_distro = 'rocky-10'
+    version_list = general_test(scylla_repo, linux_distro)
+    assert {'2025.3'} == set(version_list)
```

Comment:
not sure i follow why this test includes 3 different versions of `2025.1 ` , `branch-2025.4` and `2025.3`